action.yml

```
@@ -14,6 +14,14 @@ inputs:
   anaconda_nightly_upload_token:
     description: 'Token to upload to scientific python org'
     required: true
+  anaconda_nightly_upload_organization:
+    description: 'Organisation name to upload the wheels to'
+    required: false
+    default: scientific-python-nightly-wheels
+  anaconda_nightly_upload_labels:
+    description: 'Labels assigned to the uploaded artifacts'
```

```suggestion
    description: 'List of labels assigned to the uploaded artifacts'
```

As this could be a list it should be stated in the description.